isis/src/qisis/objs/ProjectItemModel/ProjectItemModel.cpp

```diff
@@ -912,6 +912,23 @@ namespace Isis {
   }
 
 
+  QVariant ProjectItemModel::data(const QModelIndex &index, int role) const {
+    if (index.isValid())
+      if (role == Qt::BackgroundRole || role == Qt::ForegroundRole) return QVariant();
```

Comment:
we don't really need the Qt::ForegroundRole check here. we might want to add a comment about what this check is doing as well (something like _this causes any view that uses this model directly to ignore the background colors being set through setData() with Qt::BackgroundRole. this is used so the SortFilterProxyModel can set background colors and have them displayed through attached views (the observation solve settings tree view in the JigsawSetupDialog)._